### PR TITLE
front: display electrical profile set name in scenario

### LINKED ISSUE #8002: Name of the electrical profile set chosen for the scenario isn't displayed
### What happened?

As you can see on the screenshot, when you pick an electrical profile set or a scenario, only the ID is displayed and not the name. 

This happens only in V2 not V1

image

### What did you expect to happen?

I would expect to see the name next to the infrastructure

### How can we reproduce it (as minimally and precisely as possible)?

1. Create a V2 scenario with an electrical profile set
2. In the train creation page, see that no name is displayed

### What operating system, browser and environment are you using?

- **Browser**: Firefox v115
- **OS**: Windows 11
- **Env**: osrd.dev.dgexsol.fr

### OSRD version (top right corner `Account` button > `Informations`)

803e0d1

### PR DESCRIPTION
**Pull request by clarani**
closes #8002 

After:
image

### CODE REVIEW

**Comment by SharglutDev** [comment]
Regarding your screenshot, it doesn't quite match the mockup : https://www.sketch.com/s/2fb064cd-c483-46ac-bd30-72705fd63e75/p/9935859D-9F0B-489C-9C2D-1E55C396D8DC/canvas

We should still see : "Profils électriques : FRANCE GAIA"

File: front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx [hunk 1]
```diff
@@ -115,7 +128,7 @@ const ScenarioDescription = ({
       </div>
       <div className="scenario-details-electrical-profile-set">
         {scenario.electrical_profile_set_id
-          ? scenario.electrical_profile_set_id
+          ? electricalProfileSet?.name || ''
```
Comment (Akctarus) [inline front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx]: Is it possible that an electrical profile has no name? In which case, will we ever get the || ' ' case ?

**Review by Akctarus — APPROVED** [review]
Tested, LGTM, left a question in comment

**Comment by clarani** [comment]
I modified my PR, here is what it looks like now:
image

File: front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx (hunk 1 above)
Comment (clarani) [reply front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx]: Done :white_check_mark: (I display the name + the id, so even if there is no name, an identifier is displayed :+1:

**Review by Math-R — APPROVED** [review]
Looks good ! thanks !

File: front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx (hunk 1 above)
Comment (Akctarus) [reply front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx]: Ty ! LGTM

**Pull request merged**